tests/system/python/scripts/package/readings35.py

```diff
@@ -0,0 +1,68 @@
+"""
+FogLAMP filtering for readings data
+using Python 3.5
+"""
```

Comment:
This file should kept under `tests/system/python/packages/data/` as this is prerequisite for the test. I believe we should add `readings35.py.bak` in .gitignore tests directory

Reply:
> we should add readings35.py.bak in .gitignore tests directory
If we are creating at runtime in test. we need to ensure its removal in clean up.